docker-compose-prod.yml

```diff
@@ -1,5 +1,3 @@
-version: '3.9'
```

Comment:
We don't need to have docker version mentioned as this is removed from docker attributes.